### PR TITLE
Replacing URIs with CURIEs in spec and schema

### PR DESCRIPTION
**Pull request by dzerbino**
Hello, 

This PR is meant to address #31 .

@sitag / @dbujold : I had a look through the validator and did not find any explicit validation of ontology terms, did I miss something? 

Cheers,

Daniel

### CODE REVIEW

**Comment by dzerbino** [comment]
Rebased to master (with new schema file locations)

**Comment by sitag** [comment]
@dzerbino  I didn't add any. I was hoping to piggyback off IHEC_Data_Hub validator (so we export that validation as a library function), but I don't see anything in that PR except your commits.

**Comment by zxenia** [comment]
@dzerbino What is the status of moving to CURIEs instead of URIs ? can this be merged?
@dbujold  and I discussed ontology term validation, so the question arose should we validate against CURIE or URI format first and then validate if term actually exists.

**Comment by sitag** [comment]
@zxenia @dzerbino @dbujold All this pull request seems to be for is to change URIs to CURIEs in the schema. As far as I can tell this does not adequately address https://github.com/IHEC/ihec-ecosystems/issues/31 : to address that I would expect working code to validate the CURIEs (esp. as the outcome of that issue is "This was voted on and accepted by the committee, this will be put in place for all future submissions", nothing is in place till is it's in the validator code). So can we clarify to the scope of this pull request to just be updating schemas? 

And then start a new pull request tracking work on actually validating the CURIEs. Right now I don't see where the work on actual CURIE validation is. 

Can we have some way to log what a pull request is attempting to do, so we can check that it does that, and nothing else?  

Same for #41 having validateHGNC code in validator reports pr.

**Comment by dzerbino** [comment]
Hello @sitag ,

Yes, evidently this PR was continued in PR #41 . I would say both branches should be merged simulatenously (or one rebased on top of the other, whatever you prefer)

HTH,

Daniel

File: schemas/json/experiment.json [hunk 1]
```diff
@@ -9,9 +9,9 @@
         "reference_registry_id": {"type": "string", "description": "The IHEC EpiRR ID for this dataset, only for IHEC Reference Epigenome datasets. Otherwise leave empty."},
 
         "experiment_type": {"type": "string", "description": "(Controlled Vocabulary) The assay target (e.g. ‘DNA Methylation’, ‘mRNA-Seq’, ‘smRNA-Seq’, 'Histone H3K4me1')."},
-        "experiment_ontology_uri": {"type": "string", "description": "(Ontology: OBI) links to experiment ontology information."},
+        "experiment_ontology_curie": {"type": "string", "description": "(Ontology: OBI) links to experiment ontology information."},
```
[the remark this answers is not in the document]
Comment (dzerbino) [reply schemas/json/experiment.json]: Good idea!

**Pull request merged**